cpp/src/utilities/cudf_utils.h

```diff
@@ -22,9 +23,20 @@ bool gdf_is_valid(const gdf_valid_type *valid, gdf_index_type pos) {
 		return true;
 }
 
+
+/**
+ * Calculates the size in bytes of a validity indicator pseudo-column for a given column's size.
+ *
+ * @note Actually, this is the size in bytes of a column of bits, where the individual
+ * bit-container elements are of the same size as `gdf_valid_type`.
+ *
+ * @param[in] column_size the number of elements, i.e. the number of bits to be available
+ * for use, in the column
+ * @return the number of bytes necessary to make available for the validity indicator pseudo-column
+ */
 CUDA_HOST_DEVICE_CALLABLE
-gdf_size_type gdf_get_num_chars_bitmask(gdf_size_type size) { 
-	return (( size + ( GDF_VALID_BITSIZE - 1)) / GDF_VALID_BITSIZE ); 
+gdf_size_type get_number_of_bytes_for_valid(gdf_size_type column_size) {
```

Comment:
Hmm this is tricky. We are in the process of probably changing the data type that holds the valid mask, heck maybe even the name. And the purpose of this function might change with it. If the data type holding the data changes to uint32, then what should this return? the number of uint32 elements needed or the number of bytes? Right now those two are the same, but if the underlying data type changes, it is not the same, which is why I like that the current name (get_number_of_bytes_for_valid) because it specifies that what we are getting are bytes

Reply:
IMO: 
* the current function name is awkward, overwordy English: it's unclear what "number of bytes for valid" means.
* words like "of" and "for" don't belong in API names .
* "size" in C generally means `size` in bytes. Or at least `sizeof` does. 

So (again, IMO) I think any of the following names are more clear than what you currently have: 
* `sizeof_valid_mask`
* `valid_mask_byte_size` (If you insist on having "bytes" in the name)
* `valid_mask_size`